README.md

```diff
@@ -30,6 +30,10 @@ $ yarn add @shopify/i18next-shopify
 ### Configure i18next to use this plugin
 
 ```js
+// Note: The `Intl.PluralRules` API is required for this plugin's plural handling.
+//   If your target environments do not support this API, you may need to polyfill it.
+//   This plugin's plural handling is not compatible with i18next's fallback
+//   to JSON format v3 plural handling.
```

Comment:
The below styling should produce a "Warning" dialogue with an emoji, I haven't tested it though.

````suggestion
> **Warning**
> The [`Intl.PluralRules` API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/PluralRules) is required for this plugin's plural handling.
> If your target environments do not support this API, you may need to [polyfill](https://github.com/eemeli/intl-pluralrules) it.
> This plugin's plural handling is not compatible with i18next's fallback to JSON format v3 plural handling.

```js
````

Reply:
Oh, I see. You moved it out of the comment... Anyways, it works great!